<launch>
    <node name="path_track" pkg="agv_driver" type="path_track" output="screen">
        <param name="pid_p" type="double" value="0.3"/>
        <param name="pid_i" type="double" value="0.1"/>
        <param name="pid_d" type="double" value="0.1"/>
        <param name="ave_vel" type="double" value="0.2"/>
    </node>
</launch>
